package.json

```diff
@@ -38,53 +38,60 @@
   "devDependencies": {
     "@ephox/bedrock-client": "^14.1.0",
     "@ephox/bedrock-server": "^14.1.2",
-    "@ephox/oxide-icons-tools": "^3.1.1",
+    "@ephox/oxide-icons-tools": "^4.0.0",
     "@ephox/swag": "^4.6.0",
     "@tinymce/eslint-plugin": "^2.3.0",
     "@tinymce/moxiedoc": "^0.3.0",
-    "@types/chai": "^4.3.3",
-    "@types/dompurify": "3.0.2",
+    "@types/chai": "^4.3.12",
+    "@types/dompurify": "3.0.5",
     "@types/prismjs": "^1.16.6",
-    "chai": "^4.4.1",
-    "chalk": "^4.1.0",
+    "chai": "^5.1.0",
+    "chalk": "^5.3.0",
     "changie": "^1.14.0",
     "emojilib": "^2.4.0",
     "eslint-plugin-only-warn": "^1.1.0",
     "fast-check": "^2.23.0",
-    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "fork-ts-checker-webpack-plugin": "^9.0.2",
+    "glob-parent": "^6.0.2",
     "grunt": "^1.5.3",
     "grunt-contrib-clean": "^2.0.0",
     "grunt-contrib-concat": "^2.0.0",
     "grunt-contrib-copy": "^1.0.0",
+    "grunt-contrib-cssmin": "^5.0.0",
     "grunt-contrib-symlink": "^1.0.0",
     "grunt-contrib-watch": "^1.1.0",
     "grunt-eslint": "^24.0.0",
     "grunt-nuget-pack": "^1.0.1",
     "grunt-shell": "^4.0.0",
+    "grunt-stylelint": "^0.20.1",
     "grunt-terser": "^2.0.0",
-    "grunt-webpack": "^5.0.0",
+    "grunt-webpack": "^6.0.0",
     "lerna": "^8.1.2",
-    "less-plugin-autoprefix": "^2.0.0",
+    "less": "^4.2.0",
+    "less-plugin-preset-env": "^1.2.6",
     "load-grunt-tasks": "^5.1.0",
     "moxie-zip": "0.0.4",
     "npm-run-all": "^4.1.5",
-    "patch-package": "^7.0.0",
+    "patch-package": "^8.0.0",
+    "postcss": "^8.4.39",
+    "postcss-preset-env": "^9.5.15",
     "resolve": "^1.18.1",
-    "rollup": "^3.22.0",
+    "rollup": "^4.12.1",
     "rollup-plugin-string": "^3.0.0",
-    "source-map-loader": "^4.0.1",
+    "source-map-loader": "^5.0.0",
     "string-replace-loader": "^3.0.3",
-    "stylelint": "^13.2.1",
-    "stylelint-config-standard": "^20.0.0",
-    "stylelint-order": "^5.0.0",
-    "terser": "^5.17.4",
+    "stylelint": "^16.6.1",
+    "stylelint-config-standard": "^36.0.0",
+    "stylelint-less": "^3.0.1",
+    "stylelint-order": "^6.0.4",
+    "terser": "^5.28.1",
     "ts-loader": "^9.2.6",
     "tsconfig-paths-webpack-plugin": "^4.0.1",
     "tslib": "^2.0.0",
     "twemoji": "^14.0.2",
-    "typescript": "^4.8.3",
-    "webpack": "^5.74.0",
+    "typescript": "^5.3.3",
```

Comment:
um, woah woah woah. 

Aside from the fact that you shouldn't go around upgrading every dependency in a random PR, TS5 is a big change. It's good to know that the editor still compiles, I'm sure someone said we had errors after upgrading. Maybe Microsoft fixed them since 5 launched.

- [ ] Revert any change not directly related to stylelint or grunt